src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIQuarrier.java

```diff
@@ -702,4 +704,15 @@ public BlockState getSolidSubstitution(final BlockPos ignored)
     {
         return getMainFillBlock().defaultBlockState();
     }
+
+    @Override
+    protected void triggerMinedBlock(@NotNull final BlockState blockToMine)
+    {
+        super.triggerMinedBlock(blockToMine);
+        if (IColonyManager.getInstance().getCompatibilityManager().isOre(blockToMine))
+        {
+            building.getColony().getStatisticsManager().increment(ORES_MINED);
+        }
+        building.getColony().getStatisticsManager().increment(BLOCKS_MINED);
```

Comment:
I wonder if these should be mutually exclusive.  Technically a mined ore is also a mined block, but for the purposes of showing these one after the other in a stats list I think most people would assume they're not overlapping.

Reply:
Doesn't matter a lot tbh, the difference is gonna be so large, it's barely noticeable.